src/circllhist.c

```diff
@@ -721,7 +721,7 @@ hist_approx_count_below_exclusive(const histogram_t *hist, double threshold) {
       bucket_upper = bucket_bound;
     else
       bucket_upper = bucket_bound + hist_bucket_to_double_bin_width(hist->bvs[i].bucket);
-    if(bucket_upper <= threshold)
```

Comment:
Since the buckets are defined as `[bucket_lower, bucket_upper)`,  (for values  > 0) I believe what you had earlier was correct. i.e. this needs to be `bucket_upper <=threshold`. Since all values in the bucket are strictly < threshold if `bucket_upper<=threshold`.

Reply:
For eg, if `threshold == 9`, we want the count for the bucket `[9, 10)` to be included for this method.

Reply:
The bucket is [9.0, 9.1), but yes.  Did the last commit fix that?  The bin width should be 0.1 there.  In below_excusive, we would not want it counted.  In below_inclusive we would want that counted.

Reply:
Yikes, I think inclusive and exclusive are backwards in my patch.